include/cjdb/contracts.hpp

```diff
@@ -4,10 +4,46 @@
 #ifndef CJDB_CONTRACTS_HPP
 #define CJDB_CONTRACTS_HPP
 
-#include <cstdio>
 #include <string_view>
 #include <type_traits>
 
+#ifdef _MSC_VER
+	#define CJDB_PRETTY_FUNCTION __FUNCSIG__
+	#define CJDB_FORCE_INLINE __forceinline
+#else
+	#define CJDB_PRETTY_FUNCTION __PRETTY_FUNCTION__
+	#define CJDB_FORCE_INLINE [[gnu::always_inline]] inline
+#endif // _MSC_VER
+
+#ifndef CJDB_PRINT_ERROR
+	#ifdef CJDB_USE_STDIO
```

Comment:
Please invert this: I made a (very rare) conscious decision was made to prefer `<cstdio>` over `<iostream>`. Please also minimise the amount of code in the include guards. I'd prefer two blocks with only the relevant code over one large block with duplicated code in branches.